app/views/podcast_planner/_form_side.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="card">
+  <div class="card-header-primary">
+    <h5 class="card-title">Schedule Preview</h5>
+  </div>
+  <%= turbo_frame_tag "preview", target: "_top" do %>
+    <% if @planner.dates.present? %>
+      <%= form_with(
```

Comment:
this form is what sends a `put` query to hit the `update` action to actually generate the episodes.